dart/math/Geometry.h

```diff
@@ -428,6 +428,22 @@ bool verifyRotation(const Eigen::Matrix3d& _R);
 /// all the elements are not NaN values.
 bool verifyTransform(const Eigen::Isometry3d& _T);
 
+/// Get the remainder of dividing x by y
+inline double mod(double x, double y)
+{
```

Comment:
Any reason we don't use [`std::fmod(double, double)`](http://en.cppreference.com/w/cpp/numeric/math/fmod)?